firmware/qemu/src/bin/dbg.rs

```diff
@@ -20,9 +21,11 @@ fn main() -> ! {
     let _: (i32, i32) = dbg!(x - 2, x + 2);
 
     // dbg! with zero arguments
+    #[allow(clippy::let_unit_value)]
     let _: () = dbg!();
 
     // dbg! with trailing comma
+    #[allow(clippy::double_parens)]
```

Comment:
I think this is a bug in clippy: https://github.com/rust-lang/rust-clippy/issues/9000